### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by romua**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# scafeinder
\ No newline at end of file
+# MeanApp
```
Comment (mykytamorachov) [inline README.md]: What is the name of the app?

Comment (AntonZhygalov) [reply README.md]: scafeinder

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.0.0-beta.28.3.
```
Comment (mykytamorachov) [inline README.md]: Seriously? beta version? In your package.json version is 1.3.2, be attentive when copypaste

**Review by mykytamorachov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mykytamorachov) [reply README.md]: Please use it here, not MeanApp

**Review by mykytamorachov — CHANGES_REQUESTED** [review]

**Review by mykytamorachov — APPROVED** [review]

**Pull request merged**